src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -535,7 +537,7 @@ public function postTriggerAction($uuid, Request $request)
                     $data = $repository->copyNode($uuid, $srcLocale, $webspace, $language, $userId);
                     break;
                 case 'order':
-                    $position = (int) $this->getRequestParameter($request, 'position', true);
+                    $position = (int)$this->getRequestParameter($request, 'position', true);
```

Comment:
Hm, if I set the intellij formatter to the symfony2 conventions it does that for me... I think we should stick with it then, since most people might have configured it like that.